README.md

```diff
@@ -494,6 +495,24 @@ For immidiete effect make sure to run `source ~/.bashrc`. Also you can manage co
 
 ---
 
+## Git branching workflow
+
+There are two important branches: `main` (representing stable version of the
+project) and `devel` (where most of the work happens; this is the default
+branch).
+
+- New features, and especially breaking changes, should go to the `devel` branch.
+- Bugfixes, especially ones that should be published in a bugfix release,
+  should be merged into the `main` branch.
+  - Start such bugfix branches from `main`, and make PRs to `main`.
```

Comment:
Will such PR trigger GHA?

Reply:
I think it should be done via GHA, not manually.

Reply:
PRs to `main` will trigger GHA checks. I don't think we can/should automate making the PRs with bugfixes themselves.